libs/auth-api-lib/src/lib/services/delegationScope.service.ts

```diff
@@ -144,6 +150,63 @@ export class DelegationScopeService {
     ]
   }
 
+  async findPersonalRepresentativeScopes(
+    toNationalId: string,
+    fromNationalId: string,
+  ): Promise<string[]> {
+    const apiScopes = await this.apiScopeModel.findAll({
+      where: {
+        enabled: true,
+        grantToPersonalRepresentatives: true,
+        alsoForDelegatedUser: false,
+      },
+      include: [
+        {
+          model: PersonalRepresentativeScopePermission,
+          required: true,
+          include: [
+            {
+              model: PersonalRepresentativeRightType,
+              required: true,
+              where: {
+                validFrom: {
+                  [Op.or]: { [Op.eq]: null, [Op.lt]: new Date() },
+                },
+                validTo: {
+                  [Op.or]: { [Op.eq]: null, [Op.gt]: new Date() },
+                },
+              },
+              include: [
+                {
+                  model: PersonalRepresentativeRight,
+                  required: true,
+                  include: [
+                    {
+                      model: PersonalRepresentative,
+                      required: true,
+                      where: {
+                        [Op.and]: [
+                          {
+                            nationalIdPersonalRepresentative: toNationalId,
+                          },
+                          {
+                            nationalIdRepresentedPerson: fromNationalId,
```

Comment:
4 joins, yay!

Should there be a couple of indices on these two fields to help with common queries?